setup.py

```diff
@@ -195,8 +195,8 @@ def _get_arrow_lib_as_linker_input(self):
         'azure-common<2.0.0',
         'azure-storage-blob>=12.0.0,<13.0.0',
         'boto3>=1.4.4,<2.0.0',
-        'requests<2.24.0',
-        'urllib3>=1.20,<1.26.0',
+        'requests<3.0.0',
```

Comment:
Do you still need those? I think you can safely remove them since you added both requests and urlib as vendored library as well as added the requests dependencies below, you do not need (or even should not have) any dependencies directly to requests/urllib.

Unless there is a good reason to keep them?

Reply:
Yes, when we send OCSP queries to the responders, only then we don't want to do OCSP checks.
So unfortunately we still need to keep `requests` as a dependency.

For the future we could make each PUT/GET call decide if they wanted OCSP or not.